### PR TITLE
Feature/pdu

### PR DESCRIPTION
**Pull request by trasher**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

### CODE REVIEW

File: inc/pdu.class.php [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+      echo "<textarea cols='45' rows='3' id='comment' name='comment' >".
+           $this->fields["comment"];
+      echo "</textarea></td></tr>";
+
+      $this->showFormButtons($options);
+      return true;
+   }
+
+   /*function getSearchOptionsNew() {
```
Comment (orthagh) [inline inc/pdu.class.php]: todo ?

File: inc/pdu.class.php [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+            'rand'      => $rand
+         ]
+      );
+      echo "</td></tr>\n";
+
+      echo "<tr class='tab_bg_1'>";
+      echo "<td><label for='comment'>".__('Comments')."</label></td>";
+      echo "<td colspan='3' class='middle'>";
+
+      echo "<textarea cols='45' rows='3' id='comment' name='comment' >".
```
Comment (orthagh) [inline inc/pdu.class.php]: not mandatory, but we have a function ``Html::textarea``

File: inc/pdu.class.php (hunk 1 above)
Comment (trasher) [reply inc/pdu.class.php]: Indeed, yes; I forget to do that part

File: inc/pdu.class.php (hunk 2 above)
Comment (trasher) [reply inc/pdu.class.php]: Just quickly tested... Seems that there are issues with labels (name is not part of the id, rand seems ignored, ...) :(

**Pull request merged**